starter-code/index.html

```diff
@@ -1,31 +1,64 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>Instagram</title>
-</head>
-<body>
-  Instagram
+  <head>
+    <title>Instagram</title>
+    <link rel="stylesheet" href="./css/main.css" />
+  </head>
 
-  Capture and Share the World's Moments
+  <body>
+    <section>
+      <!--IMAGE IPHONE-->
+      <div class="imageIphone">
+        <img src="./images/phones.png" alt="images of iphone" />
+      </div>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+      <!--RIGHT CONTAINER-->
+      <div class="right-container">
+        <!--HEADER-->
+        <header class="header">
+          <div><img src="./images/brand.png" alt="" /></div>
```

Comment:
No need to put the image inside a < div > here, you can do it only if you have for example 2 images. For just one, you can leave it alone :) 
And don't forget to write something inside your alt tag too :)
```
<img src="./images/brand.png" alt="brand image" />
```